worlds/bomb_rush_cyberfunk/Rules.py

```diff
@@ -0,0 +1,1043 @@
+        lambda state: brink_terminal_mesh(state, player, limit, glitched))
+    set_rule(multiworld.get_location("Brink Terminal: Eclipse joins the crew", player),
+        lambda state: current_chapter(state, player, 3))
+    if photos:
+        set_rule(multiworld.get_location("Brink Terminal: Behind glass Polo", player),
+            lambda state: camera(state, player))
+    
+    # millennium mall
+    set_rule(multiworld.get_location("Millennium Mall: Glass cylinder CD", player),
+        lambda state: inline_skates(state, player, movestyle))
+    set_rule(multiworld.get_location("Millennium Mall: Trick challenge reward", player),
+        lambda state: millennium_mall_challenge1(state, player))
+    set_rule(multiworld.get_location("Millennium Mall: Slide challenge reward", player),
+        lambda state: millennium_mall_challenge2(state, player))
+    set_rule(multiworld.get_location("Millennium Mall: Fish challenge reward", player),
+        lambda state: millennium_mall_challenge3(state, player))
+    set_rule(multiworld.get_location("Millennium Mall: Score challenge reward", player),
+        lambda state: millennium_mall_challenge4(state, player))
+    set_rule(multiworld.get_location("Millennium Mall: Atrium BMX gate BMX", player),
+        lambda state: bmx(state, player, movestyle))
+    set_rule(multiworld.get_location("Millennium Mall: Shine joins the crew", player),
+        lambda state: current_chapter(state, player, 4))
+    set_rule(multiworld.get_location("Millennium Mall: DOT.EXE joins the crew", player),
+        lambda state: current_chapter(state, player, 4))
+    
+    # pyramid island
+    set_rule(multiworld.get_location("Pyramid Island: BMX gate BMX", player),
+        lambda state: bmx(state, player, movestyle))
+    set_rule(multiworld.get_location("Pyramid Island: Score challenge reward", player),
+        lambda state: pyramid_island_challenge1(state, player))
+    set_rule(multiworld.get_location("Pyramid Island: Score challenge 2 reward", player),
+        lambda state: pyramid_island_challenge2(state, player))
+    set_rule(multiworld.get_location("Pyramid Island: Quarter pipe challenge reward", player),
+        lambda state: pyramid_island_challenge3(state, player))
+    set_rule(multiworld.get_location("Pyramid Island: Shortcut glass CD", player),
+        lambda state: inline_skates(state, player, movestyle))
+    set_rule(multiworld.get_location("Pyramid Island: Maze outfit", player),
+        lambda state: skateboard(state, player, movestyle))
+    if not glitched:
+        add_rule(multiworld.get_location("Pyramid Island: Rise joins the crew", player),
+            lambda state: camera(state, player))
+    set_rule(multiworld.get_location("Pyramid Island: Devil Theory joins the crew", player),
+        lambda state: current_chapter(state, player, 5))
+    if photos:
+        set_rule(multiworld.get_location("Pyramid Island: Polo pile 1", player),
+            lambda state: camera(state, player))
```

Comment:
```suggestion
        set_rule(multiworld.get_location("Pyramid Island: Polo pile 1", player),
            camera)
```
These  all could  be shortened like this, as you're just wrapping the function.